test/python/fe_api/test_grouped_gemm_wrapper_memo.py

```
@@ -0,0 +1,189 @@
+    # forward the caller's value on every call, hit or miss. current_stream is per-call
+    # state, not a cache dimension.
+    NOT_CACHE_DIMENSIONS = {"current_stream", "linear_offset"}
+
+    checked = []
+    for module_info in pkgutil.walk_packages(cutedsl.__path__, cutedsl.__name__ + "."):
+        if not module_info.name.endswith(".api"):
+            continue
+        try:
+            module = importlib.import_module(module_info.name)
+        except Exception:
+            continue
```

Ruff (0.16.1)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)